contracts/interchain-token-service/src/msg.rs

```diff
@@ -53,6 +53,17 @@ pub enum ExecuteMsg {
     EnableExecution,
 }
 
+#[cw_serde]
+pub enum ChainStatusFilter {
+    Frozen,
+    Active,
+}
+
+#[cw_serde]
+pub struct ChainConfigFilter {
```

Comment:
```suggestion
pub struct ChainFilter {
```

might not necessarily be filtering on the chain config, could filter on something else in the future